Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN apt-get install -y ca-certificates libvips37 libxml2-dev \
     && make install \
     && ldconfig
 RUN mkdir /etc/vip
+RUN ln -s /usr/lib/libvips.so.42 /usr/lib/libvips.so.38
```

Comment:
Ok, I have it on the roadmap to update to a much newer version of `libvips` as well as using that different Go binding for it since it'll give us transparency support in PNGs and the ability to resize GIFs, among other things.